CHANGELOG.rst

```diff
@@ -1,5 +1,18 @@
 .. _changelog:
 
+0.40.2
```

Comment:
Thank you @olevski !
I just wonder if this section should be after `0.40.1` and before `0.41.0`, WDYT?

Reply:
Ah yeah that is what I mean to do. My bad.